src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/Licence.kt

```diff
@@ -79,6 +79,7 @@ data class Licence(
   val spoDiscussion: String? = null,
   val vloDiscussion: String? = null,
   val reasonForVariation: String? = null,
+  val reasonForReferral: String? = null,
```

Comment:
So going with only one reason initially?

Reply:
That's the plan. Then evolve so `licence_history` becomes `licence_event` and we can add different event types to reflect the `reasonForVariation` and `reasonForReferral` conversation between COM and ACO. Still room for discussion over on how that works.